Dockerfile

```diff
@@ -13,45 +8,52 @@ RUN apk add --no-cache --update git
 COPY go.mod ./
 COPY go.sum ./
 RUN go mod download
-#RUN go mod vendor
 
-#COPY . .
 COPY ./cmd ./cmd
 COPY ./pkg ./pkg
 
 RUN CGO_ENABLED=0 go build -o ./prover ./cmd/prover/prover.go
 
 
-# Build an prover image
-FROM node:14
+# Main image
+FROM node:16
 
 ENV APP_USER=app
 ENV APP_UID=1001
-ENV DOCKER_GID=999
 
-RUN apt-get update && apt-get install -y ca-certificates
-RUN mkdir -p /home/$APP_USER
-RUN adduser -u $APP_UID $APP_USER && chown -R $APP_USER:$APP_USER /home/$APP_USER
-RUN addgroup --system --gid ${DOCKER_GID} docker
-RUN addgroup ${APP_USER} docker
-RUN rm rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
+
+RUN adduser -u $APP_UID $APP_USER --disabled-password --gecos "First Last,RoomNumber,WorkPhone,HomePhone"
```

Comment:
--gecos "First Last,RoomNumber,WorkPhone,HomePhone" ??

Reply:
This is needed for `adduser` commant to pass without warnings.